File: ADS - 3º CICLO/L. de Programação - DP/EXERCICIOS/ENTRADA_SAIDA/src/Exercicios/Soma.java
package Exercicios;
import java.util.Scanner;

public class Soma{
    public static void main( String[] args ){
        Scanner sc = new Scanner( System.in );

        System.out.println("INFORME UM VALOR: "); int A = sc.nextInt();
        System.out.println("INFORME OUTRO VALOR: "); int B = sc.nextInt();
        int soma = A + B;

        System.out.println("------------------------------");
        System.out.println("1º VALOR: " + A ); System.out.println("2º VALOR: " + B );
        System.out.print("SOMA: " + soma );
    }
}

File: ADS - 4º CICLO/Linguagem de Programação - PHP/ads_lp_php/Atividades/Exercicios/soma.php
<?php
    $X = 10;
    $Y = 20;

    $soma = 10 + ( ++$X );
    $subs = $soma - ( $Y-- );

    echo "VALOR X : " . $X . "<br>";
    echo "VALOR Y : " . $Y . "<br>";
    echo "SOMA : " . $soma . '<br>';
    echo "SUBS : " . $subs . '<br>';
?>